File: L1/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>
#include <ctype.h>

int main(void)
{
    
    int i, j, k, height;
    
    do
    { 
        
        height = get_int("Height: ");
    
    }
    while (height < 1 || height > 8);
    
    for (i = 0; i < height; i++)
    {
        
        //LEFT
        int num_blankspace = height - i - 1;

        for (k = 0; k < num_blankspace; k++)
        {
            printf(" ");
        }
        
        for (k = 0; k < i + 1; k++)
        {
            printf("#");
        }
            
        
        // two middle spaces
        printf("  ");
        
        //RIGHT
        for (k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        
        //next line
        printf("\n");
    }
}

File: L6/mario/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break

for i in range(n):
    # Left
    for j in range(n-i-1):
        print(" ", end="")
    for j in range(i+1):
        print("#", end="")

    # Middle two spaces
    print("  ", end="")

    # Right
    for j in range(i+1):
        print("#", end="")

    # Next line
    print()